src/main/java/dynamok/sink/DynamoDbSinkTask.java

```diff
@@ -94,6 +114,11 @@ public void put(Collection<SinkRecord> records) {
                     }
                 }
             }
+        } catch (LimitExceededException | ProvisionedThroughputExceededException e) {
+            log.debug("Batch write of {} records failed with Limit/Throughput Exceeded exception; backing off",
```

Comment:
Since it can be individual put requests now if the `batch.size=1` default config is used, and the number of records may be less than the configured batch size, consider updating the log line to be simply:
```
log.debug("Write failed with Limit/Throughput Exceeded exception; backing off", e);
```
Also, is it worth logging at info/warn rather than debug?

Nice improvement catching those exceptions and retrying BTW, unrelated to this PR but I am not a stickler for that :)